src/BacklightSimulator.tsx

```diff
@@ -1,10 +1,8 @@
 import { useCallback, useEffect, useRef } from 'react';
+import { computeBacklightFrame } from './colorManipulation';
+import { regionConvolution } from './colorManipulation';
```

Comment:
if we're importing from the same file, we should combine our imports for code cleanliness